gamemode/modules/base/sh_createitems.lua

```diff
@@ -5,7 +5,7 @@ local plyMeta = FindMetaTable("Player")
 -----------------------------------------------------------
 local function declareTeamCommands(CTeam)
     local k = 0
-    for num,v in pairs(RPExtraTeams) do
+    for num,v in ipairs(RPExtraTeams) do
```

Comment:
`ipairs` can't deal with holes in tables. Again a micro optimisation that you'll have to be **VERY** careful with. Depending on how the disabling of default job is implemented, there might be holes in `RPExtraTeams`. There may also be addons that just set some index of `RPExtraTeams` to `nil`, which would break this code.

Reply:
You already use ipairs in a lot of situations especially for this table.